CMake/FindIBMMASS.cmake

```diff
@@ -1,14 +1,14 @@
 # Simple file to find IBM MASS (if available)
 INCLUDE( CheckCXXSourceCompiles )
 
-MESSAGE("Looking for IBM mass libraries")
+MESSAGE(STATUS "Looking for IBM MASS libraries")
 # Finding and setting the MASS_INCLUDE_DIRECTORIES
 set(SUFFIXES include)
 find_path(MASS_INCLUDE_DIRECTORIES name "mass.h" HINTS ${MASS_ROOT}
   PATH_SUFFIXES ${SUFFIXES})
-if (MASS_INCLUDE_DIRECTORIES-NOTFOUND)
+if (NOT MASS_INCLUDE_DIRECTORIES)
```

Comment:
This seem to be the right way to check.

Reply:
I copy pasted that from somewhere but if it does still work I think you're right here.